client/packages/requisitions/src/RequestRequisition/DetailView/columns.ts

```diff
@@ -107,6 +107,13 @@ export const useRequestColumns = () => {
         width: 150,
         getSortValue: rowData => rowData.requestedQuantity,
       },
+      // Should only be visible if linked requisition has authorisationStatus (also authorisation comment)
+      {
+        key: 'approvedQuantity',
```

Comment:
:nit should be right aligned

Reply:
Can this possible be done in front end changes PR, as a follow up ?